packages/cli/src/runner/file-command-runner.ts

```diff
@@ -30,67 +33,83 @@ export class FileCommandRunner implements CommandRunner {
         @inject(ReportGenerator) private readonly reportGenerator: ReportGenerator,
         @inject(ReportDiskWriter) private readonly reportDiskWriter: ReportDiskWriter,
         @inject(JsonSummaryReportGenerator) private readonly jsonSummaryReportGenerator: JsonSummaryReportGenerator,
+        @inject(HtmlSummaryReportGenerator) private readonly htmlSummaryReportGenerator: HtmlSummaryReportGenerator,
         @inject(ConsoleSummaryReportGenerator) private readonly consoleSummaryReportGenerator: ConsoleSummaryReportGenerator,
         private readonly fileSystemObj: typeof fs = fs,
     ) {}
 
     public async runCommand(scanArguments: ScanArguments): Promise<void> {
-        const spinner = new Spinner(`Running scanner...\n`);
+        const spinner = new Spinner(`Running scanner... %s \t`);
```

Comment:
Running scanner is so confusing now, it prints too many lines and after the scan is done, it looks weird if the user would look at it

Reply:
![spiner](https://user-images.githubusercontent.com/45858632/81355184-80d7dc80-9082-11ea-8886-6c80f9d9d306.gif)

Reply:
try the latest version. I had fixed this. spinner was printing in new line before due to the \n.

Reply:
check the gif file in the PR